lib/ansible/modules/system/filesystem.py

```diff
@@ -16,21 +16,22 @@
 author:
 - Alexander Bulimov (@abulimov)
 module: filesystem
-short_description: Makes file system on block device
+short_description: Makes filesystem
 description:
-  - This module creates file system.
+  - This module creates filesystem.
 version_added: "1.2"
 options:
   fstype:
     choices: [ btrfs, ext2, ext3, ext4, ext4dev, lvm, reiserfs, xfs ]
     description:
-    - File System type to be created.
+    - FileSystem type to be created.
```

Comment:
As much as i doubt which form is better above .. camel case is not in the running at all

Reply:
Done